core/src/ledger_cleanup_service.rs

```diff
@@ -16,9 +17,10 @@ use std::time::Duration;
 // - A validator to download a snapshot from a peer and boot from it
 // - To make sure that if a validator needs to reboot from its own snapshot, it has enough slots locally
 //   to catch back up to where it was when it stopped
-pub const DEFAULT_MAX_LEDGER_SLOTS: u64 = 270_000;
-// Remove a fixed number of slots at a time, it's more efficient than doing it one-by-one
-pub const DEFAULT_PURGE_BATCH_SIZE: u64 = 256;
+pub const DEFAULT_MAX_LEDGER_SHREDS: u64 = 270_000 * 60;
```

Comment:
Is it safe to assume 60/shreds a slot at full TPS? The goal is to keep `270_000` slots around even at full capacity right?

Reply:
yea, this is just the default, the limit can be adjusted with --limit-ledger-slots. The 270k number is 4x the original number calculated for 50, so I think this number will be sufficient. I can put the math there though.

Reply:
Ok, I added some math based on experimental results.